docs/content/tutorials/helm/index.md

```diff
@@ -0,0 +1,647 @@
+---
+type: docs
+title: "Tutorial: Use Helm to run your first app"
+linkTitle: "Helm application"
+weight: 200
+description: "Take a tour of Radius by updating an existing Helm chart to add Radius support."
+categories: "Tutorial"
+tags : ["kubernetes", "helm"]
+---
+
+This tutorial will teach you the following about Radius:
+
+* How to use update a Helm chart to include its resources in a Radius application
+* How to use features like Recipes and Connections within Kubernetes YAML or Helm.
+< image src="diagram.png" alt="Diagram of the application and its resources" width="500px" >
+
+## Prerequisites
+
+- [rad CLI]({{< ref "installation#step-1-install-the-rad-cli" >}})
+- [Helm](https://helm.sh/docs/intro/install/)
+- [Setup a supported Kubernetes cluster]({{< ref "/guides/operations/kubernetes/overview#supported-clusters" >}})
+
+## Step 1. Clone and open the sample code
+
+{{< tabs "Codespace" "Local machine" >}}
+
+{{% codetab %}}
+
+It's easy and free to get up and running with a Radius Codespace in GitHub. Spin up a Radius environment in seconds with the following link:
+
+[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/radius-project/samples)
+
+Once launched you should already have the application cloned locally. Use the terminal to navigate to the `./demo/` directory:
+
+```bash
+cd ./samples/demo
+```
+
+{{% /codetab %}}
+{{% codetab %}}
+Use the terminal to clone the `samples` repository locally and navigate to the `./samples/demo` directory:
+
+```bash
+git clone https://github.com/radius-project/samples.git
+cd ./samples/samples/demo
+```
+{{% /codetab %}}
+
+{{< /tabs >}}
+
+
+## Step 2. Initialize Radius
+
+Initialize Radius. For this example, answer **NO** when asked whether to set up an application: 
+
+```bash
+rad init
+```
+
+> Select 'No' when prompted to create an application.
+
+## Step 3. Understand and deploy the application
+
+1. Navigate to the `./Chart` folder and browse its contents. This contains a Helm chart for the application that you will modify.
```

Comment:
rebased this PR -  https://github.com/radius-project/samples/pull/620/files and moved the chart templates inside the demo folder to match with the instructions